src/Spout/Common/Escaper/ODS.php

```diff
@@ -26,13 +26,13 @@ public function escape($string)
             // 'ENT_DISALLOWED' ensures that invalid characters in the given document type are replaced.
             // Otherwise control characters like a vertical tab "\v" will make the XML document unreadable by the XML processor
             // @link https://github.com/box/spout/issues/329
-            $replacedString = htmlspecialchars($string, ENT_NOQUOTES | ENT_DISALLOWED, 'UTF-8');
+            $replacedString = htmlspecialchars($string, ENT_QUOTES | ENT_DISALLOWED, 'UTF-8');
         } else {
             // We are on hhvm or any other engine that does not support ENT_DISALLOWED.
             //
-            // @NOTE: Using ENT_NOQUOTES as only XML entities ('<', '>', '&') need to be encoded.
-            //        Single and double quotes can be left as is.
-            $escapedString =  htmlspecialchars($string, ENT_NOQUOTES, 'UTF-8');
+            // @NOTE: We must use ENT_QUOTES because single and double quotes may be used in spreadsheet names.
```

Comment:
Can you please change that note to something more global like this: "Using ENT_QUOTES as XML entities ('<', '>', '&') as well as double quotes (for values of XML attributes) need to be encoded." ?